src/plugins/Debug.ts

```diff
@@ -44,15 +60,21 @@ class Debug extends GenericPlugin {
 
   private insertMessage(baseMessage: string): void {
     const currentDate = new Date();
-
     const hour = padTime(currentDate.getHours());
     const minutes = padTime(currentDate.getMinutes());
     const milliseconds = padTime(currentDate.getMilliseconds());
-
     const message = `[EVENT] ${baseMessage} at ${hour}:${minutes}:${milliseconds}`;
 
     insertElement('p', { style: MESSAGE_STYLE }, this.debugOverlay, message);
   }
+
+  private insertMetadata(metadata: googletag.events.SlotRenderEndedEvent): void {
```

Comment:
It would be really nice to be able to just do a `JSON.stringify(event, null, '\t' or '\n')` but insert element just dumps strings into the html regardless of formatting. I can fix it if thats what we want but it will still have shape to it with `{` and '}`. I think that displaying it this way makes more sense.

Reply:
I like what you've done here. I agree that the JSON stringify would be simpler but the output is ugly!